### ISSUE TITLE
Test the EStaRs

### ISSUE THREAD

**Issue by sidneylok** [AguaClara/1-LPS#5]
Troubleshoot and test

**Comment by ericamarroquin** [comment]
According to the team that worked with the EStaRS filter this past summer (2017), the filter has been water tested and it is water-tight. Our next step is to add sand to the filter and get water flowing through. @monroews is there any specific way we should add sand, or can we just fill until the upper-most inlet?

**Comment by monroews** [comment]
I'd begin by checking all filter seals (top and bottom) and then fill it with water to test it for water tightness. If it passes that test, then proceed...

I'd want to check to make sure that the slotted and orifice pipes are held firmly in place, are able to handle the force of backwash (equal to the excess head [head in addition to the head required to lift the sand] that is available to initialize fluidization of the sand bed). Take the max water level in the inlet tank - the level of the siphon discharge and minus the depth of sand. That is the excess head that is available. Then take that excess head and calculate a pressure and turn that into a force based on the filter area. That is the force that the slotted pipe manifolds have to withstand. 

If the inlet and outlet pipes pass this analysis, then confirm that you have good seals on the bottom of the filter and fill it with sand to about 5 cm above the top inlet.

Load the filter with water from the bottom to drive the air up and out of the filter.

**Comment by ericamarroquin** [comment]
EStaRS filter was tested today (3/1/18). There are a few leaks in the welds that need to be fixed under the exit tank. Bottom of the trunk needs to be examined because more leaks are suspected, even when tightened fully with hose clamps. 

Will test for water tightness again on Tuesday, and hopefully fix the remaining issues with the welds.

**Comment by SungMinKim2795** [comment]
3/6 Moved the EStaRS back into the AguaClara lab to test because the drain isn't operational for this week (and possibly longer).

**Comment by SungMinKim2795** [comment]
3/8 Took apart the entrance tank of the EStaRS. There was significantly more water inside of it so draining took some time. Looked at the welds inside and saw that they weren't very good. Will reinforce them tomorrow (3/9) to see if we need to epoxy it.

**Comment by SungMinKim2795** [comment]
3.9 Meeting with Monroe. Ordered a 12" fernco cap for the bottom of the EStaRs.

**Comment by ericamarroquin** [comment]
Order for 12" Fernco cap was cancelled because it was out of stock with the vendor, and would take too much time to get to us. 

EStaRS filter was reassembled after additional welding was done to ensure watertightness. We plan on testing the filter on Tuesday (3/22) to see if additional welds are needed or if we should use epoxy. 

Force calculations are finished. The [force analysis](https://github.com/AguaClara/1-LPS/blob/master/EStaRS_Force_Analysis.md) can be found here. The total force due to backwash is 679.8 newtons (152.8 lbf). That means each manifold would have 91.7 (21.83 lbf) exerted on it. I can't really visualize that in my head - @monroews is this a reasonable force?

**Comment by monroews** [comment]
Your calculations seem reasonable. This is the equivalent of half a bucket of water. So you could somehow apply 10 kg of weight to the center of the top manifold and see what happens.

**Comment by sidneylok** [comment]
3/22 We tried to test the top manifold by filling a bucket with 10L of water and placing it on top of manifold. The manifold was ferncoed to a longer pipe and the pipe was clamped to one end of a lab bench so that the manifold hung from the edge of the bench. The manifold was able to withstand the weight. Tim placed the manifold back into the body.

**Comment by monroews** [comment]
This is excellent. Thus this connection system should work and you can proceed to test the filter!

**Comment by sidneylok** [comment]
@monroews Would it be possible to move the EStaRS to the grad lab? It is currently obstructing the path to the eye wash station.

**Comment by monroews** [comment]
Yes. You can work with @WPennock. There is room in the grad lab.

**Comment by sidneylok** [comment]
3/27 Erica ordered a fernco and pipe stub for the 1-1/4" entrance pipe stub. The drain is still down... We moved the EStaRS to the grad lab.

**Comment by sidneylok** [comment]
3/29 The fernco, washers, and pipe came in. However, the entrance stub is actually 1.5" so what we got doesn't fit the bill. We had to order fernco reducer for 1.5" pipe size to 1.25" pipe size for current testing and another 1.5" straight pipe fernco for when when we test with the full plant.

**Comment by SungMinKim2795** [comment]
4/10 Unable to find the top for the filter. Will be remaking the top this week.

**Comment by ericamarroquin** [comment]
EStaRS Filter is now assembled, water tight, and ready to be filled with sand. The filter will be filled with 0.45-0.55 mm sand just past the first inlet. Next, I am getting dosages and pumps set up to do clay testing.

**Comment by ericamarroquin** [comment]
Realizing space might get a little cramped with both the EStaRS and 1 LPS being tested at the same time. There is only one source and one available set of pumps and turbidimeters. @monroews any suggestions? Is there anywhere I can move the filter to test? Its still on its cart so it can be put anywhere. I need to test ASAP, as does the 1 LPS, and it would take too much time to share.

**Comment by monroews** [comment]
The goal is to test the filter with the rest of the 1 L/s plant. We don't have another space for testing and testing it separately doesn't meet our goals. We want to know if the whole system can run and produce clean water.
If you need another turbidimeter so that you can measure raw, settled, and filtered water turbidity, that is easy to daisy chain (using 3 different ID numbers on the turbidimeters) and I believe we have extra turbidimeters in the lab.

**Comment by ericamarroquin** [comment]
After filling the filter with sand, the bottom of the filter failed. The heavy leak could not be fixed by adjusting the Fernco or shifting the filter base. I met with Paul to discuss better ways of fabricating the EStaRS that can be implemented soon. A schematic will be added shortly.
